docs/sites.yml

```diff
@@ -5383,3 +5383,87 @@
   built_by: Nikodem Deja
   built_by_url: https://nikodemdeja.pl
   featured: false
+- title: manuvel.be
+  url: https://www.manuvel.be
+  main_url: https://www.manuvel.be
+  description: >
+    Cycling themed café coming this april in Sint Niklaas, Belgium. Has funky css-grid trickery!
+  categories:
+    - Café
```

Comment:
Could we perhaps use Coffee here too? I know it's not exact, but given that Coffee is added in other entry we would end up with Café and Coffee categories each with single entry